src/mapkit.cs

```diff
@@ -2292,4 +2337,261 @@ interface MKZoomControl
+	{
+		[Export ("initWithCoordinate:")]
+		[DesignatedInitializer]
+		NativeHandle Constructor (CLLocationCoordinate2D coordinate);
+
+		[Export ("initWithMapItem:")]
+		[DesignatedInitializer]
+		NativeHandle Constructor (MKMapItem mapItem);
+
+		[Export ("coordinate")]
+		CLLocationCoordinate2D Coordinate { get; }
+
+		[NullAllowed, Export ("mapItem")]
+		MKMapItem MapItem { get; }
+
+		[Export ("isCancelled")]
+		bool IsCancelled { get; }
+
+		[Export ("isLoading")]
+		bool IsLoading { get; }
+
+		[Async]
+		[Export ("getSceneWithCompletionHandler:")]
+		void GetScene (Action<MKLookAroundScene, NSError> completionHandler);
+
+		[Export ("cancel")]
+		void Cancel ();
+	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof (NSObject))]
+	interface MKLookAroundSnapshot
+	{
+		[Export ("image")]
+		UIImage Image { get; }
+	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof (NSObject))]
+	interface MKLookAroundSnapshotOptions
+	{
+		[NullAllowed, Export ("pointOfInterestFilter", ArgumentSemantic.Copy)]
+		MKPointOfInterestFilter PointOfInterestFilter { get; set; }
+
+		[Export ("size", ArgumentSemantic.Assign)]
+		CGSize Size { get; set; }
+
+		[NoMac, iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+		[Export ("traitCollection", ArgumentSemantic.Copy)]
+		UITraitCollection TraitCollection { get; set; }
+	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MKLookAroundSnapshotter
+	{
+		[Export ("initWithScene:options:")]
+		[DesignatedInitializer]
+		NativeHandle Constructor (MKLookAroundScene scene, MKLookAroundSnapshotOptions options);
+
+		[Async]
+		[Export ("getSnapshotWithCompletionHandler:")]
+		void GetSnapshot (Action<MKLookAroundSnapshot, NSError> completionHandler);
+
+		[Export ("cancel")]
+		void Cancel ();
+
+		[Export ("isLoading")]
+		bool IsLoading { get; }
+	}
+
+	interface IMKLookAroundViewControllerDelegate { }
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+#if NET
+	[Protocol, Model]
+#else
+	[Protocol, Model (AutoGeneratedName = true)]
+#endif
+	[BaseType (typeof (NSObject))]
+	interface MKLookAroundViewControllerDelegate
+	{
+		[Export ("lookAroundViewControllerWillUpdateScene:")]
+		void WillUpdateScene (MKLookAroundViewController viewController);
+
+		[Export ("lookAroundViewControllerDidUpdateScene:")]
+		void DidUpdateScene (MKLookAroundViewController viewController);
+	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof (UIViewController))]
+	interface MKLookAroundViewController : NSSecureCoding, NSCoding
+	{
+		[Export ("initWithScene:")]
+		[DesignatedInitializer]
+		NativeHandle Constructor (MKLookAroundScene scene);
+
+		[Export ("initWithNibName:bundle:")]
+		[DesignatedInitializer]
+		NativeHandle Constructor ([NullAllowed] string nibNameOrNil, [NullAllowed] NSBundle nibBundleOrNil);
```

Comment:
We do not need to add OrNil in the name because is marked as nullable AND we call it null in c#
```suggestion
		NativeHandle Constructor ([NullAllowed] string nibName, [NullAllowed] NSBundle nibBundle);
```